easybuild/easyconfigs/o/OptiX/NVIDIA-OptiX-6.0.0-CrayGNU-19.09.eb

```diff
@@ -0,0 +1,22 @@
+# contributed by Jean Favre (CSCS)
+easyblock = "Tarball"
+
+name = 'OptiX'
```

Comment:
The name of the dir might have to be renamed here as well.

Reply:
I would rename the recipe instead as just `OptiX-6.0.0-CrayGNU-19.09.eb`, since the `name` of the software appearing inside the recipe is just `Optix`: what do you think, @jfavre ?